### DISCUSSION TITLE
Can't get it to work?

### DISCUSSION THREAD

**Discussion by BluesFanUK** [AAndyProgram/SCrawler#151] [General]
I have .NET Framework 4.6 (later) installed, downloaded the zip, put it in a separate folder, ran the program, added a Twitter account, but when I go to download it just downloads the settings. What am I doing wrong?

EDIT 1 - I have since added a cookie to the Twitter setting (I hope this is a one time thing only...), and it downloaded a profile, but it was missing numerous images - the log says:
Some posts didn't download. You can see them in the 'Missing posts' form - there isn't a Missing Posts form, or at least it's well hidden.

The next user I added I got the same as the original issue, downloads the profile image and settings, that's it. I'm perplexed.

**Comment by AAndyProgram** [comment]
Missing posts: https://github.com/AAndyProgram/SCrawler/wiki#missing-posts
Settings: https://github.com/AAndyProgram/SCrawler/wiki/Settings#site-settings

> but it was missing numerous images

Could you post the profile URL?

**Reply by BluesFanUK** [reply]
Thanks - re Missing posts, it was showing two reddit links for me. I opened both posts where there were images, SCrawler's download button isn't picking them up for some unknown reason. In any case I wanted to try fresh.

The issues i'm having seem to be literally anything today, any random profile I try. I've followed all instructions and apparently have a .NET environment greater than the requirement - so steps below:

```
1. Download the latest build, saved to a location on my largest HDD.
2. Run program, creates the environment
3. In my browser go to my Twitter account, export Netscape cookie to a file named twitter.txt in the SCrawler directory (using cookie.txt, a Firefox addon).
4. In SCrawler Go to Settings > Twitter > Open cookies pencil icon > Import > Ctrl+O > select Twitter.txt
5. Click Add, auto detects user, press save.
6. Right Click > Download
```

It then goes through a process of downloading profile with a quick notification to say everything's downloaded, and all i end up with is a folder for settings and a thumbnail (if i'm lucky).

Have I done something wrong?


EDIT - I re-added the cookie again and tested on PennyCakez twitter account (low post rate). It downloaded 58 images... but it's missing the first couple of her most recent images, so even when it works it's not downloading everything.

**Reply by AAndyProgram** [reply]
> SCrawler's download button isn't picking them up for some unknown reason.

Select them, then click download.

> EDIT - I re-added the cookie again and tested on PennyCakez twitter account (low post rate). It downloaded 58 images... but it's missing the first couple of her most recent images, so even when it works it's not downloading everything.

I've fully downloaded her (https://twitter.com/PennyCakez/media) media. I see a bug that gifs are downloaded in static, but they are still present. Sorry.

Is there data in the LOG?

> It then goes through a process of downloading profile with a quick notification to say everything's downloaded, and all i end up with is a folder for settings and a thumbnail (if i'm lucky).

A quick notification that all data has been downloaded without actually downloading can be only if the profile is empty or you don't have cookies set.

If the profile contains only retweets, make sure that the `Get user media only` checkbox ([user settings](https://github.com/AAndyProgram/SCrawler/wiki#add-user)) is unchecked. Unfortunately, you will need to download these profiles again ([how to do it](https://github.com/AAndyProgram/SCrawler/wiki#redownload-user)).
